src/main/java/me/totalfreedom/totalfreedommod/command/Command_toggle.java

```diff
@@ -77,6 +78,12 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
             return true;
         }
 
+        if (args[0].equals("explosivearrows"))
+        {
+            toggle("Explosive Arrows feature now is", ConfigEntry.MAKE_ARROW_EXPLOSIVES);
```

Comment:
grammatical error, should be "Explosive arrows are now" if it's `true` or `false` on the config entry.

Reply:
So what i should put?

Reply:
i just told you right there